keyboards/dztech/dz60rgb/keymaps/mechmaster48/README.md

```diff
@@ -0,0 +1,15 @@
+# Mechmaster48 keymap
```

Comment:
```suggestion
Copyright 2021 MechMaster48 <YOUR@EMAIL-ADDRESS>
 
 This program is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 2 of the License, or
 (at your option) any later version.
 
 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 # Mechmaster48 keymap
```

Reply:
I have added the GPL copyright notice.